src/aooptionsdialog.cpp

```diff
@@ -723,7 +737,7 @@ void AOOptionsDialog::save_pressed()
   configini->setValue("stickyeffects", ui_stickyeffects_cb->isChecked());
   configini->setValue("stickypres", ui_stickypres_cb->isChecked());
   configini->setValue("customchat", ui_customchat_cb->isChecked());
-
+  configini->setValue("automatic_logging_enabled", ui_casing_log_cb->isChecked());
```

Comment:
"ui_casing_log_cb" should be "ui_log_cb"

Reply:
oops, I missed that one. Will do